SECURITY_FEATURES.md

```diff
@@ -151,3 +153,30 @@ The policy in Bottlerocket has the following objectives:
 
 The policy is currently aimed at hardening the OS against persistent threats.
 Future enhancements to the policy will focus on mitigating the impact of OS vulnerabilities, and protecting containers from other containers.
+
+### Kernel lockdown in integrity mode
+
+Bottlerocket enables Lockdown in "integrity" mode by default on most variants.
+
+Lockdown is a Linux Security Module (LSM) that blocks certain actions which could compromise the Linux kernel.
+As with SELinux, even processes that run as root with full capabilities are subject to these restrictions.
+
+Certain variants such as `*-nvidia` need to load unsigned kernel modules at runtime.
+This is prohibited by the "integrity" mode, but required for the hardware to work as expected.
+On these variants, Lockdown is set to "none" instead.
+
+### Secure Boot enabled
+
+Bottlerocket enables Secure Boot for all new variants on platforms that support UEFI boot.
```

Comment:
nit: I'm OK with this right now, but what would a “new variant” be? Should we instead use the first version that enabled secure boot?

Reply:
> nit: I'm OK with this right now, but what would a “new variant” be? Should we instead use the first version that enabled secure boot?

I'd rather document that in the release notes, as we did for the new Lockdown defaults in [v1.1.0](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.1.0).

Otherwise this content becomes less "evergreen" - if we mention `*-k8s-1.28` variants here and a year from now they've been pruned from the tree, does that mean Secure Boot isn't enabled? Conversely, if we mention a specific Bottlerocket version rather than variants, it could be interpreted as saying that everything supports Secure Boot as of that version.